examples/src/main/java/com/expediagroup/sdk/xap/examples/scenarios/lodging/shopping/AvailabilityCalendarsScenario.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2024 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.sdk.xap.examples.scenarios.lodging.shopping;
+
+import com.expediagroup.sdk.xap.examples.scenarios.XapScenario;
+import com.expediagroup.sdk.xap.examples.services.lodging.shopping.AvailabilityCalendarsService;
+import com.expediagroup.sdk.xap.models.AvailabilityCalendarResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This example demonstrates how to use Availability Calendar api with simple search.
+ */
+public class AvailabilityCalendarsScenario implements XapScenario {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(AvailabilityCalendarsScenario.class);
+
+  private final AvailabilityCalendarsService availabilityCalendarService =
+      new AvailabilityCalendarsService();
+
+  @Override
+  public void run() {
+    // This example returns the availability of each day for a range of dates for given Expedia
+    // lodging properties.
+
+    LOGGER.info("============ Running Lodging Availability Calendars Scenario ============");
```

Comment:
"Running <scenario name>" should match the class name to make it easily trackable.